pkg/oktaApiAuth/api_types.go

```diff
@@ -10,6 +10,19 @@
 
 package oktaApiAuth
 
+import "errors"
+
+var (
+	errContinue        = errors.New("continue")
+	errPushFailed      = errors.New("Push MFA failed")
+	errTOTPFailed      = errors.New("TOTP MFA failed")
+	errMFAUnavailable  = errors.New("No MFA factor available")
+	errMFARequired     = errors.New("MFA required")
+	errUserLocked      = errors.New("User locked out")
+	errPasswordExpired = errors.New("User password expired")
+	errEnrollNeeded    = errors.New("Needs to enroll")
```

Comment:
Maybe leave the errors that are used only for tests in the test file so that we don't expect to actually find/use them in the code itself.

Reply:
Hum, actually only `errTOTPFailed` is only used in tests, i had the feeling to be more consistent having it with the others, WDYT ?